piws/views/logout.py

```diff
@@ -0,0 +1,105 @@
+from cubicweb.web.views.sessions import InMemoryRepositorySessionManager
```

Comment:
#! /usr/bin/env python
##########################################################################
# NSAp - Copyright (C) CEA, 2013
# Distributed under the terms of the CeCILL-B license, as published by
# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
# for details.
##########################################################################